packages/app/src/specs/SpecsList.vue

```diff
@@ -153,7 +153,9 @@
     v-model="isLoginOpen"
     :gql="props.gql"
     :utm-medium="loginUtmMedium"
-    @loggedin="refreshPage"
+    show-connect-button-after-login
+    @loggedin="handleLoggedin"
+    @connect-project="isProjectConnectOpen = true"
```

Comment:
I am surprised this works - should this not be a callback? `() => isProjectConnectOpen = true`?

Reply:
I've always just thought of these inline expressions as idiomatic Vue (like this [`count++` example](https://vuejs.org/guide/essentials/event-handling.html#inline-handlers) in the docs) - same syntax is valid for `onclick` attribute in HTML. 

We handle events this way plenty already (eg `@show-spec-pattern-modal="showSpecPatternModal = true"`, and several similar things). There are a couple reasons I prefer the non-callback version but they are all pretty marginal, we could standardize on something else.